test/util_test.go

```diff
@@ -193,7 +193,7 @@ func startDqlite(ctx context.Context, tb testing.TB, dir string, listener *instr
 
 	driver, err := dqlite.NewDriver(ctx, &dqlite.DriverConfig{
 		DB: database.NewPrepared(db),
-	})
+	}, nil)
```

Comment:
Why passing nil here? I think we have the `app` so we should pass it down.

Reply:
Oh, right. missed it.